apps/web/app/helpers/date.ts

```diff
@@ -96,6 +96,20 @@ export const formatDateTimeString = (dateTimeString?: string) => {
 	return '';
 };
 
+export const formatDateString = (dateTimeString?: string) => {
```

Comment:
@paradoxe35 

I think it is better to use `moment(dateTimeString).format('DD MMM YYYY')` here.